Packs/MaintenanceUpdateNotifications/Playbooks/playbook-Send_Maintenance_Email.yml

```diff
@@ -0,0 +1,129 @@
+contentitemexportablefields:
+  contentitemfields:
+    propagationLabels:
+    - all
+id: Send Maintenance Email
+inputs: []
+name: Send Maintenance Email
+outputs: []
+quiet: true
+starttaskid: "0"
+tasks:
+  "0":
+    continueonerrortype: ""
+    id: "0"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "1"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 23af3943-7959-4c20-8e78-316c87d239f5
+      iscommand: false
+      name: ""
+      version: -1
+      description: ''
+    taskid: 23af3943-7959-4c20-8e78-316c87d239f5
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+  "1":
+    continueonerrortype: ""
+    id: "1"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      cc:
+        simple: ${incident.emailccto}
+      htmlBody:
```

Comment:
is it possible to customize that with input? Assuming that some customers would like to make modifications to the HTML body according to their needs... so may I suggest converting this into a PB input?

Reply:
We don't want to harm the integrity of the notification by allowing customers to modify the post body. If this is a feature that is requested we can add it in the future, but for now we just need to get this data in front of the customers as is. :)